### ISSUE TITLE
gtk+3 build error: ld: cannot find -lcups

### ISSUE THREAD

**Issue by tseemann** [Linuxbrew/homebrew-core#4247]
```
==> Upgrading gtk+3
==> Downloading https://linuxbrew.bintray.com/bottles/gtk+3-3.22.21.x86_64_linux.bottle.tar.gz
#                                                                          2.3%
curl: (18) transfer closed with 27549629 bytes remaining to read
Error: Failed to download resource "gtk+3"
Download failed: https://linuxbrew.bintray.com/bottles/gtk+3-3.22.21.x86_64_linux.bottle.tar.gz
Warning: Bottle installation failed: building from source.
==> Downloading https://download.gnome.org/sources/gtk+/3.22/gtk+-3.22.21.tar.xz
Already downloaded: /home/linuxbrew/.cache/Homebrew/gtk+3-3.22.21.tar.xz
==> ./configure --enable-debug=minimal --prefix=/home/linuxbrew/.linuxbrew/Cellar/gtk+3/3.22.21 --disable-glibtest --enable-introspecti
==> make install
Last 15 lines from /home/linuxbrew/.cache/Homebrew/Logs/gtk+3/02.make:
  CC       gtkcupssecretsutils.lo
  CCLD     libprintbackend-cups.la
/home/linuxbrew/.linuxbrew/bin/ld: cannot find -lcups
/home/linuxbrew/.linuxbrew/bin/ld: cannot find -lgssapi_krb5
/home/linuxbrew/.linuxbrew/bin/ld: cannot find -lkrb5
/home/linuxbrew/.linuxbrew/bin/ld: cannot find -lk5crypto
/home/linuxbrew/.linuxbrew/bin/ld: cannot find -lcom_err
collect2: error: ld returned 1 exit status
make[3]: *** [libprintbackend-cups.la] Error 1
make[3]: Leaving directory `/tmp/gtk+3-20170914-445-1a46q6/gtk+-3.22.21/modules/printbackends/cups'
make[2]: *** [install-recursive] Error 1
make[2]: Leaving directory `/tmp/gtk+3-20170914-445-1a46q6/gtk+-3.22.21/modules/printbackends'
make[1]: *** [install-recursive] Error 1
make[1]: Leaving directory `/tmp/gtk+3-20170914-445-1a46q6/gtk+-3.22.21/modules'
make: *** [install-recursive] Error 1

HOMEBREW_VERSION: 1.3.1-1-g9c79fed
ORIGIN: https://github.com/Linuxbrew/brew.git
HEAD: 9c79fedcd105d6c11e14303681f1598587a13b46
Last commit: 31 hours ago
Core tap ORIGIN: https://github.com/Linuxbrew/homebrew-core
Core tap HEAD: 4305acd1fa5794941f945646806f47d55fcfe684
Core tap last commit: 3 hours ago
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_REPOSITORY: /home/linuxbrew/.linuxbrew
HOMEBREW_CELLAR: /home/linuxbrew/.linuxbrew/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://linuxbrew.bintray.com
CPU: 72-core 64-bit haswell
Homebrew Ruby: 2.4.1 => /home/linuxbrew/.linuxbrew/Cellar/ruby/2.4.1_2/bin/ruby
Clang: 5.0 build
Git: 2.14.1 => /home/linuxbrew/.linuxbrew/bin/git
Perl: /home/linuxbrew/.linuxbrew/bin/perl => /home/linuxbrew/.linuxbrew/Cellar/perl/5.26.0/bin/perl
Python: /home/linuxbrew/.linuxbrew/opt/python/libexec/bin/python => /home/linuxbrew/.linuxbrew/Cellar/python/2.7.13_1/bin/python2.7
Ruby: /home/linuxbrew/.linuxbrew/bin/ruby => /home/linuxbrew/.linuxbrew/Cellar/ruby/2.4.1_2/bin/ruby
Java: openjdk version "1.8.0_144"
Kernel: Linux 3.10.0-693.2.1.el7.x86_64 x86_64 GNU/Linux
OS: Red Hat Enterprise Linux Server release 7.4 (Maipo) (Maipo)
Host glibc: 2.17
/usr/bin/gcc: 4.8.5
glibc: 2.23
gcc: 5.3.0
xorg: 20170115
```

**Comment by iMichka** [comment]
See #497. Need to investigate when that option went away. Maybe some merge broke it lately.

**Comment by iMichka** [comment]
Got it, this was `gtk+`, and you are installing `gtk+3`.
I opened a pull request to fix that: #4367

**Comment by iMichka** [comment]
Fix merged. Please test.
